cmd/vela-git/main.go

```diff
@@ -67,6 +67,13 @@ func main() {
 
 		// Build Flags
 
+		&cli.StringFlag{
+			EnvVars:  []string{"PARAMETER_BRANCH", "GIT_BRANCH", "VELA_PULL_REQUEST_SOURCE", "VELA_BUILD_BRANCH"},
```

Comment:
We specify the `VELA_PULL_REQUEST_SOURCE` environment variable first intentionally.

This environment variable is ONLY set for `pull_request` events.

This ensures the CLI library we use will try to set this value from that variable first.

In the event this plugin runs for a non `pull_request` event, we then source from `VELA_BUILD_BRANCH`.

If all else fails, we have a fallback value of `master` to mirror the `build.ref` flag:

 https://github.com/go-vela/vela-git/blob/b41373c735ff8abb4802a67610b3b9770c81fdf3/cmd/vela-git/main.go#L82-L88